### DISCUSSION TITLE
Is there any way to get window position (x, y) coords?

### DISCUSSION THREAD

**Discussion by ahrvoje** [wezterm/wezterm#6023] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22 x86_64-pc-windows-msvc

### Ask your question!

Is there any interface to get window's position (x, y) coords?

There is a window:set_position(x, y), and I need the opposite operation, something like window:get_position(). It doesn't have to be a method, just about any interface to get the coords would suffice.

**Comment by Alien-lv426** [comment]
Have you found a way of doing this? I've been searching too but haven't come up with anything.

**Reply by ahrvoje** [reply]
No, not yet.

**Reply by ahrvoje** [reply]
FYI

"Add get_position lua method for window" https://github.com/wez/wezterm/pull/3006 [still Open]

**Comment by Alien-lv426** [comment]
Thanks. I'm not sure that ever made it into wezterm. All I get is 

`attempt to call a nil value (method 'get_position')`

Does it work for you?

**Reply by Risinggoblin** [reply]
it's still an open PR and hasn't been merged yet. You can subscribe to the PR to get an update once it's merged
